ui/components/app/flask/copyable/copyable.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../../ui/box';
+import IconCopied from '../../../ui/icon/icon-copied';
+import IconCopy from '../../../ui/icon/icon-copy';
+import Typography from '../../../ui/typography';
+import {
+  ALIGN_ITEMS,
+  BORDER_RADIUS,
+  COLORS,
+  JUSTIFY_CONTENT,
+  OVERFLOW_WRAP,
+  FLEX_DIRECTION,
+} from '../../../../helpers/constants/design-system';
+import { useCopyToClipboard } from '../../../../hooks/useCopyToClipboard';
+
+export const Copyable = ({ text }) => {
+  const [copied, handleCopy] = useCopyToClipboard();
+  return (
+    <Box
+      backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
+      alignItems={ALIGN_ITEMS.STRETCH}
+      justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
+      borderRadius={BORDER_RADIUS.SM}
+      paddingLeft={4}
+      paddingRight={4}
+      paddingTop={2}
+      paddingBottom={2}
+    >
+      <Typography
+        color={COLORS.TEXT_ALTERNATIVE}
+        marginRight={2}
+        overflowWrap={OVERFLOW_WRAP.ANYWHERE}
+      >
+        {text}
+      </Typography>
```

Comment:
Let's make text inside the `<Copyable />` container same font-size (_Body-MD_) as the body text outside of it; `16px` or `0.875rem`